pyserverforindy/grpc_server/anoncreds_service.py

```diff
@@ -2,7 +2,11 @@
 import os
 import sys
 
-from indy import anoncreds as indy_anoncreds
```

Comment:
no major changes in anoncres_service. Just add IndyError handler for future changes